packages/opentelemetry-api-metrics/package.json

```diff
@@ -53,12 +53,12 @@
     "access": "public"
   },
   "peerDependencies": {
-    "@opentelemetry/api": "^1.0.0"
+    "@opentelemetry/api": "^1.0.1"
```

Comment:
@open-telemetry/javascript-maintainers  What is our guideline regarding the API version to use in peer dependencies?

Do we stick on ^1.0.0 to avoid problems if users have pinned 1.0.0?
=> in this case we should tell bot about this.

Or do we move to latest available?

Reply:
I think we should bump our version. If a user has pinned 1.0.0 then it will still work, the user will just get a warning from NPM.

Reply:
ok fine. at least in some places `^1.0.1` is already used in contrib.